app/src/release/res/values/google_maps_api.xml

```diff
@@ -0,0 +1,18 @@
+<resources>
+    <!--
+    TODO: Before you run your application, you need a Google Maps API key.
+
+    To get one, follow this link, follow the directions and press "Create" at the end:
+
+https://console.developers.google.com/flows/enableapi?apiid=maps_android_backend&keyType=CLIENT_SIDE_ANDROID&r=26:94:57:FE:DE:11:18:E3:2B:5E:32:21:9E:D5:DD:71:CA:16:CC:6E%3Borg.fossasia.openevent.fragments
+
+    You can also add your credentials to an existing key, using this line:
+    26:94:57:FE:DE:11:18:E3:2B:5E:32:21:9E:D5:DD:71:CA:16:CC:6E;org.fossasia.openevent.fragments
+
+    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
+    string in this file.
+    -->
+    <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">
+        YOUR_KEY_HERE
```

Comment:
Nah it's okay I will fix that, you commit a working version of Maps for
Debug. Once it's okay lets merge it.

On Thu, Jun 11, 2015 at 6:03 PM, Manan Wason <notifications@github.com>
wrote:

> In app/src/release/res/values/google_maps_api.xml
> <https://github.com/fossasia/open-event-android/pull/27#discussion_r32206250>
> :
>
> > +<resources>
> > +    <!--
> > +    TODO: Before you run your application, you need a Google Maps API key.
> > +
> > +    To get one, follow this link, follow the directions and press "Create" at the end:
> > +
> > +https://console.developers.google.com/flows/enableapi?apiid=maps_android_backend&keyType=CLIENT_SIDE_ANDROID&r=26:94:57:FE:DE:11:18:E3:2B:5E:32:21:9E:D5:DD:71:CA:16:CC:6E%3Borg.fossasia.openevent.fragments
> > +
> > +    You can also add your credentials to an existing key, using this line:
> > +    26:94:57:FE:DE:11:18:E3:2B:5E:32:21:9E:D5:DD:71:CA:16:CC:6E;org.fossasia.openevent.fragments
> > +
> > +    Once you have your key (it starts with "AIza"), replace the "google_maps_key"
> > +    string in this file.
> > +    -->
> > +    <string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">
> > +        YOUR_KEY_HERE
>
> This is the release key file. Should I add the same key here as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fossasia/open-event-android/pull/27/files#r32206250>.
>



-- 
Mohit Singh KANWAL (Mr)

[image: Facebook] <http://facebook.com/mohitkanwal>  [image: Twitter]
<http://twitter.com/mohitkanwal>